httpclient5/src/main/java/org/apache/hc/client5/http/ssl/SSLConnectionSocketFactory.java

```diff
@@ -266,13 +266,13 @@ public Socket createLayeredSocket(
             sslsock.setEnabledCipherSuites(TlsCiphers.excludeWeak(sslsock.getEnabledCipherSuites()));
         }
 
-        if (this.log.isDebugEnabled()) {
-            this.log.debug("Enabled protocols: {}", Arrays.asList(sslsock.getEnabledProtocols()));
-            this.log.debug("Enabled cipher suites:{}", Arrays.asList(sslsock.getEnabledCipherSuites()));
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Enabled protocols: {}", Arrays.asList(sslsock.getEnabledProtocols()));
+            LOG.debug("Enabled cipher suites:{}", Arrays.asList(sslsock.getEnabledCipherSuites()));
```

Comment:
Now that we're using slf4j interpolation, logging the `String[]` directly without wrapping should be fine. Missing a space after the colon.